keras2onnx/_builtin.py

```diff
@@ -924,6 +961,43 @@ def convert_tf_reshape(scope, operator, container):
                               desired_shape=shape_value)
 
 
+@converter_func(TYPES.ScatterNd)
+def convert_tf_scatter_nd(scope, operator, container):
+    if operator.target_opset < 11:
+        raise ValueError("ScatterNd op is not supported for opset = " + str(operator.target_opset))
```

Comment:
To stay consistent w/ other functions (i.e. line 532), should be "... not supported for opset < 11"

Reply:
GreaterEqual = Less + Not